### PR TITLE
Archie/dwt rebrand/2448

### PR DESCRIPTION
**Pull request by archiepeterjarvis**
# Set splash screen to DWT on DWT machines

[Jira Ticket](https://yetitoolsmartsoftware.atlassian.net/browse/YTSS-2448)
- [x] Ready for review

## Checklist
- [x] I have completed a self review
- [ ] I have updated the [spreadsheet](https://docs.google.com/spreadsheets/d/1277nPOKtBFOk_kCo870_zA460-qlw_kGnYlLPwtnadE/edit#gid=0)
- [x] I have updated the jira ticket
- [x] I have added the relevant labels to this PR
- [ ] I have updated documentation (if applicable)
- [ ] I have run the unit tests suite and they pass
- [ ] I have updated the [change log](https://docs.google.com/document/d/1t3s6lQuUcug1Fj8E_i2gBJNvXN0-funwBArG2hsia6k/edit)

## Description
I've changed model_detector to model_manager, and added functions for setting up the machine as DWT machine, including making dwt.txt & setting the splash screen.

## Notes
Yet to be tested on console 7, as I don't currently have one. Tested on console XL though!

## Dependencies for merge
None

### Visual Test
- [ ] Not applicable
- [ ] Own computer
- [ ] Console 7"
- [x] Console 10"

### Function Test
- [ ] Not applicable
- [ ] Own computer
- [ ] Console 7"
- [x] Console 10"

### Unit Tests
- [x] Not applicable
- [ ] Completed

## Screenshots (if applicable)

### CODE REVIEW

File: src/asmcnc/comms/model_manager.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+def is_machine_drywall():
+    # () -> bool
+    """
+    Checks for the existence of the dwt.txt file.
+    :return: True if the file exists, False otherwise.
+    """
+    return os.path.exists(DWT_FILE_PATH)
+
+
+def set_machine_drywall(is_drywall):
```
Comment (Masc999) [inline src/asmcnc/comms/model_manager.py:22]: What's your idea when should this be called? I assume sometime in final test?

File: src/main.py [hunk 2]
```diff
@@ -86,7 +83,6 @@
 from asmcnc.skavaUI import screen_rebooting # @UnresolvedImport
 from asmcnc.skavaUI import screen_job_feedback # @UnresolvedImport
 from asmcnc.skavaUI import screen_job_incomplete # @UnresolvedImport
-from asmcnc.skavaUI import screen_powercycle_alert # @UnresolvedImport
```
Comment (Masc999) [inline src/main.py:89]: why removing two screens?

Comment (archiepeterjarvis) [reply src/main.py:89]: I just optimised imports with pycharm, they aren't actually used.

File: src/asmcnc/comms/model_manager.py (hunk 1 above)
Comment (archiepeterjarvis) [reply src/asmcnc/comms/model_manager.py:22]: Yeah, I thought it could be called from I assume the button that final test operator would press

Comment (archiepeterjarvis) [reply src/asmcnc/comms/model_manager.py:22]: Yes it's tested on console XL. I don't have a 7 inch console right now, but it's the same.

**Comment by Masc999** [comment]
There are conflicts in main.py. Did you pull master again?

File: src/asmcnc/comms/model_manager.py (hunk 1 above)
Comment (archiepeterjarvis) [reply src/asmcnc/comms/model_manager.py:22]: I did sudo reboot and it actually shows the updated splash screen on shutdown as well as startup.

**Pull request merged**